packages/contracts/.addresses.json

```diff
@@ -0,0 +1,6 @@
+{
+    "5": {},
+    "11155111": {
+        "arbitrator": "0x96073897873796d1950B1B04Fe2Ead8E0CA34914"
```

Comment:
Should we add the reality v3 contract addresses here?
These ones [here](https://github.com/carrot-kpi/reality-eth-v3-oracle-template/pull/14/files#diff-6a1e81dd44ca581c7e5d70e30e0f1cd5b373c92aacec21955d4770749b437d3aL9).